types/src/indexer/indexer_db_reader.rs

```diff
@@ -10,16 +10,29 @@ use crate::{
         state_value::StateValue,
         table::{TableHandle, TableInfo},
     },
-    transaction::{AccountOrderedTransactionsWithProof, Version},
+    transaction::{AccountOrderedTransactionsWithProof, ReplayProtector, Version},
 };
 use anyhow::Result;
+use aptos_crypto::HashValue;
+use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Copy, Eq, PartialEq)]
 pub enum Order {
     Ascending,
     Descending,
 }
 
+// Question[Orderless]: Do we need any more information here? How about gas_used and block timestamp?
+// Question[Orderless]: As this struct is stored in the DB, do changes to this struct break the DB?
+// Question[Orderless]: What's the right file to define this struct?
```

Comment:
guess you are gonna use this type in the API? I'd put it under types/src/transaction/mod.rs

Reply:
Done.